components/post_view/post_message_view/post_message_view.test.tsx

```diff
@@ -62,10 +62,8 @@ describe('components/post_view/PostAttachment', () => {
     test('should match snapshot, on edited post', () => {
         const props = {...baseProps, post: {...post, edit_at: 1}};
         const wrapper = shallow(<PostMessageView {...props}/>);
-        const instance = wrapper.instance() as PostMessageView;
```

Comment:
I understand we cannot any longer test this like this. Any way we can add snapshots or tests to check again the edit functionality?

Reply:
I will find a way to add tests again.